src/pybamm/experiment/step/step_termination.py

```diff
@@ -67,9 +70,19 @@ def get_event(self, variables, step):
         """
         See :meth:`BaseTermination.get_event`
         """
+        operator = self.operator
+        if operator is None:
+            expr = abs(variables["Current [A]"]) - self.value
+            string = f"abs(Current [A]) < {self.value} [A] [experiment]"
+        elif operator == ">":
+            expr = self.value - variables["Current [A]"]
+            string = f"Current [A] > {self.value} [A] [experiment]"
+        elif operator == "<":
+            expr = variables["Current [A]"] - self.value
+            string = f"Current [A] < {self.value} [A] [experiment]"
```

Comment:
It is good to end if-elif chains with an else of some sort so that future devs know nothing was accidentally left off

Reply:
In this case it won't be possible to reach the `else`, so what do you recommend doing about coverage

Reply:
I guess we could make the None an else